cypress/support/parent-commands.ts

```diff
@@ -79,6 +79,13 @@ declare namespace Cypress {
      * - on click of header, the changelog content is visible.
      */
     checkChangelogRevealer(): void;
+
+    /**
+     * Asserts that:
+     * - parsed search string has expected keys that map to the correct values
+     * - uses [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams)
+     */
+    verifyExpectedQuery(queryString: string, expectedParams: Record<string, string | boolean>): void;
```

Comment:
Do you still need this command for this changeset?

Reply:
Hmm no, I do not. Good catch. Will remove 👍